docs(elastic-metal): flex ipv6 MTA-1824 (#519)
scaleway/docs-content: compute/elastic-metal/how-to/configure-ipv6-hypervisor.mdx

---
meta:
  title: How to configure IPv6 virtual machines on Proxmox and ESXi on Elastic Metal
  description: This page explains how to configure IPv6 virtual machines on Proxmox and ESXi on Elastic Metal
content:
  h1: How to configure IPv6 virtual machines on Proxmox and ESXi on Elastic Metal
  paragraph: This page explains how to configure IPv6 virtual machines on Proxmox and ESXi on Elastic Metal
dates:
  validation: 2022-04-01
  posted: 2022-04-01
---

<Message type="requirement">

  - You have an account and are logged into the [Scaleway console](https://console.scaleway.co/m)
  - You have [created](/dedibox/dedicated-servers/how-to/order-a-server) a Dedibox and installed a hypervisor (Proxmox or ESXi) on it
  - You have ordered a [flexible IPv6](/compute/elastic-metal/how-to/order-fleible-ip/)
  - You have generated a [virtual MAC address](/dedibox-network/ip-failover/concepts/#virtual-mac-address)

</Message>

> When you install your server with a virtualization solution like VMware vSphere Hypervisor (ESXi) or Proxmox you can create multiple virtual machines on the physical server and configure them to use fleible IPv6 addresses.

Reviewer: ```suggestion
When you install your server with a virtualization solution like VMware vSphere Hypervisor (ESXi) or Proxmox, you can create multiple virtual machines on the physical server and configure them to use flexible IPv6 addresses.
```

> You can find below, examples for the configuration of the network interfaces on different distributions inside a virtual machine:

Reviewer: ```suggestion
Below are some examples for the configuration of the network interfaces on different distributions inside a virtual machine:
```
